custom_components/zwave_mqtt/entity.py

```diff
@@ -135,19 +142,36 @@ def __init__(self, values):
         self.values = values
         self.options = values._options
         self.values._entity = self
+        self._dispatcher_callbacks = []
 
     @callback
     def value_changed(self, value):
         """Call when the value is changed."""
         if value.value_id_key in (v.value_id_key for v in self.values if v):
             self.async_schedule_update_ha_state()
 
+    @callback
     def value_added(self):
         """Handle a new value for this entity."""
 
+    @callback
+    def instance_updated(self, new_status):
+        """Call when the instance status changes."""
+        self.async_schedule_update_ha_state()
```

Comment:
Use `self.async_write_ha_state` instead. We'll write the state in the same event loop iteration then.

Reply:
OK, thanks for the insight. Will be fixed